app/views/comments/settings.html.erb

```diff
@@ -1,28 +1,6 @@
 <% title "Comment Settings" %>
```

Comment:
I wasn't sure how to access this view to preview what these changes look like... @benhalpern @ludwiczakpawel can you enlighten me?

Reply:
https://d.pr/v/UFtLZH